src/2d/shallow/filpatch.f90

```diff
@@ -327,20 +326,17 @@ recursive subroutine filrecur(level,nvar,valbig,aux,naux,t,mx,my, &
                                                              + eta2 * slope(2,i_coarse,j_coarse)
                     h_fine = max(eta_fine - aux(1,i_fine + nrowst - 1, j_fine + ncolst - 1), 0.d0)
 
-                    if (variable_eta_init) &
+                    if (variable_eta_init) then
```

Comment:
How did this even compile?!

Reply:
It was an old-fashioned one-line fortran if test,
    if (condition) statement
that was split over two lines, but it confused me too when I saw it, so I decided to make it clearer.

Reply:
Oh, good point.